<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandWindow</class>
 <widget class="QMainWindow" name="CommandWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommandWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All files</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listPages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>256</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>View the page</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputViewPage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>511</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemovePage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove 
selected page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetPage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get 
selected page</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputWritePage">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>90</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUploadPage">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>510</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Upload it</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Write a page</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePageName">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>450</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>430</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Page name:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
